src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -128,7 +129,7 @@
 	private final BiomeDictionaryTypeListField biomeDictionaryTypes = new BiomeDictionaryTypeListField(mcreator);
 	private final DefaultFeaturesListField defaultFeatures = new DefaultFeaturesListField(mcreator);
 
-	private final DataListEntry.Dummy noparent = new DataListEntry.Dummy("No parent");
+	private final DataListEntry.Dummy noparent = new DataListEntry.Dummy(L10N.t("elementgui.biome.no_parent"));

```

Comment:
This one must not be translated because the name of entry is used for data storage and in templates in if statements. Same with any other dropdown.